backend/routes/remedies.js

```diff
@@ -123,4 +172,24 @@ router.delete('/:id', catchAsynch(async (req, res) => {
     return res.status(200).send(deletedRating);
 }));
 
+
+//save remedy as favorite:
+router.put('/:id/save', catchAsynch(async (req, res) => {
+    const { id } = req.params;
+    //const product = await Medicals.findById(id);
+    const userTest = "64151b8670662285f3b36c13";
```

Comment:
I would put all the hardcoded values top to the of the file.it is also helpful if you format your code with a linter like prettier